### DISCUSSION TITLE
Pyright can't resolve import on neovim even if the pythonPath is set correctly.

### DISCUSSION THREAD

**Discussion by BBerabi** [microsoft/pyright#3828] [General]
**Describe the bug**
I have installed ncoc language server and pyright. In general, it works, but pyright can not resolve the third-party import statements. 
I opened coc_setting.json via CocConfig command and defined the `python.pythonPath` variable and I also activated the environment in my terminal. 

My coc_setting.json is below:

```
{
          "semanticTokens.filetypes": ["c", "cpp", "python"],
          "clangd.path": "~/.config/coc/extensions/coc-clangd-data/install/14.0.3/clangd_14.0.3/bin/clangd",
          "python.linting.mypyEnabled": true,
          "python.pythonPath": "/home/berkay_berabi_snyk_io/autofix/deepcode/.venv/devenv/bin/python"

}
```

I also tried the command `CocCommand workspace.showOutput` Pyright and it resolves to the python in my environment. Still, I get the error `reportMissingImports`

**Screenshots or Code**
![Screenshot from 2022-08-13 19-13-59](https://user-images.githubusercontent.com/77981442/184504132-6d2e8414-343e-4896-9b54-5256a690c903.png)

What is missing in my configuration or setup? Why does it not work?

**Comment by erictraut** [comment]
Converting question to a discussion topic since this isn't a bug report.

**Reply by erictraut** [reply]
Can you provide the log output? That will probably provide some clues.

You can also enable verbose log output by adding a pyrightconfig.json file with the contents `{ "verboseOutput": true }`.
